src/variations.js

```diff
@@ -90,39 +90,6 @@ window.VARIATIONS = {
     },
   },
 
-  "FB2L2": {
```

Comment:
I think you removed the wrong one. We do need FB2L2.

Reply:
omg I did, sorry